src/components/Products.js

```diff
@@ -22,41 +22,68 @@ const Products = ({products, setProducts}) => {
 
     useEffect(fetchProducts, []); 
 
+    const searchProducts = (product, text) => {
+        text = text.toLowerCase();
+        const {name} = product;
+        for (const field of [name]) {
+            if(field.toLowerCase().includes(text)) {
+                return true;
+            }
+        }
+    }
+
+    const filteredProducts = searchTerm ? products.filter(product => searchProducts(product, searchTerm)) : products;
+
+    const addToCart = (status, userId) => {
+        console.log(currentUser)
+        console.log("Add to Cart was pushed");
+        axios.post("/api/orders", {status, userId})
+            .then(res => { 
+                console.log("Adding item to order", res)
```

Comment:
curious if we still need these logs?

Reply:
will remove these in final product, still being used for debugging.